src/Sylius/Bundle/UserBundle/Context/CustomerContext.php

```diff
@@ -24,27 +25,58 @@ class CustomerContext implements CustomerContextInterface
     /**
      * @var SecurityContextInterface
      */
-    private $securityContext;
+    protected $securityContext;
+
+    /**
+     * @var SessionInterface
+     */
+    protected $session;
 
     /**
      * @param SecurityContextInterface $securityContext
+     * @param SessionInterface $session
      */
-    public function __construct(SecurityContextInterface $securityContext)
-    {
+    public function __construct(
+        SecurityContextInterface $securityContext,
```

Comment:
Can you replace this deprecated functionality with proper ones (token storage & authentication manager)?